Scripts/Player/PlayerController.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerController : MonoBehaviour
+{
+    private PlayerStats playerStats;
+    
+    private void OnTriggerEnter(Collider other)
+    {
+
+        if (other.CompareTag("Pool"))
+        {
+            Debug.Log("girdi");
+            playerStats = GetComponent<PlayerStats>();
+            GetComponent<PlayerMovement>().MoveDown();
+        }
+    }
```

Comment:
PlayerStats ve PlayerMovement'ı Awake methodunda tek seferde alabilirsiniz. Her trigger enter olduğunda kendi üzerindeki component'ları GetComponent yapmak gereksiz.

```suggestion
private PlayerStats playerStats;
private PlayerMovement playerMovement;

void Awake()
{
    playerStats = GetComponent<PlayerStats>();
    playerMovement = GetComponent<PlayerMovement>();
}
```